### PR TITLE
node docs

### PR DESCRIPTION
**Pull request by mssimi**

### CODE REVIEW

File: docs/rector_rules_overview.md [hunk 1]
```diff
@@ -4861,7 +4861,7 @@ Remove null coalescing operator ??=
 ### `DowngradeTypedPropertyRector`
 
 - class: [`Rector\DowngradePhp74\Rector\Property\DowngradeTypedPropertyRector`](/rules/downgrade-php74/src/Rector/Property/DowngradeTypedPropertyRector.php)
-- [test fixtures](/rules/downgrade-php74/tests/Rector/Property/DowngradeTypedPropertyRector/Fixture)
+- [test fixtures](/rules/downgrade-php74/tests/Rector/Property/NoDocBlockDowngradeTypedPropertyRector/Fixture)
```
Comment (mssimi) [inline docs/rector_rules_overview.md:4864]: @TomasVotruba I don't know why is this still happening.

Comment (TomasVotruba) [reply docs/rector_rules_overview.md:4864]: I think it's a path matching bug in `dump-rectors` command.

The end path of both files is the same, so the slash need to be added to regular expression or path matching or whatever is used there to find a test case

Comment (mssimi) [reply docs/rector_rules_overview.md:4864]: I can, it's order issue.

Comment (mssimi) [reply docs/rector_rules_overview.md:4864]: Here: #4260

**Comment by mssimi** [comment]
I have added some snippets, I am taking break for now. Will continue with more later, but this can get merged imo.

**Comment by TomasVotruba** [comment]
Looks good, mergin first wave :+1: 

Btw, we usually prefix the PR titles with `[type]`, e.g. `[Docs]`. Then we generate changelog from titles, also descriptive titles are helpful for searching of old PRs for debugging.

This one would be named e.g.:

`[Docs] Add new node samples`

**Pull request merged**